src/test/kotlin/com/example/Employee/controllers/CompanyNewsControllerTest.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.Employee.controllers
+
+import com.example.Employee.models.CompanyNews
+import com.example.Employee.services.CompanyNewsService
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.jupiter.api.Test
+import org.amshove.kluent.`should be equal to`
+
+internal class CompanyNewsControllerTest{
+    private val companyNewsService= mockk<CompanyNewsService>()
+    private val companyNewsController=CompanyNewsController(companyNewsService)
+
+    @Test
+    fun `should add news`(){
+        every { companyNewsService.addNews(fakeCompanyNews) }returns Unit
```

Comment:
Please check if you need this stubbing.

Reply:
It is throwing error saying no answer provided for companyNewsService.addNews
<img width="1440" alt="Screenshot 2022-05-27 at 9 16 14 AM" src="https://user-images.githubusercontent.com/102017428/170625562-72fa729c-d875-48e2-b5af-51d8195bbc61.png">